core/src/main/java/hudson/Launcher.java

```diff
@@ -450,8 +450,14 @@ public ProcStarter writeStdin() {
 
         /**
          * Starts the new process as configured.
+         *
+         * @throws RuntimeException can be thrown if {@link #cmds()} is empty
```

Comment:
"Can" be thrown? When would it not be thrown?

Reply:
Depending on the Launcher implementation. DummyLauncher will never throw it. Is it worth clarifying this? or can we just generalize and say that "will be thrown if cmds() is empty"?